### PR TITLE
Initial commit of the openATTIC Manager module

### PR DESCRIPTION
**Pull request by LenzGr**
Create the directory and basic module initialization.

Add a `README.rst`

Signed-off-by: Lenz Grimmer <lgrimmer@suse.com>

### CODE REVIEW

**Comment by rjfd** [comment]
@LenzGr the `__init__.py` you created should be renamed to `module.py`. Then you should add a file `__init__.py` with the following content:

```
from module import *  # NOQA
```

**Review by rjfd — CHANGES_REQUESTED** [review]
https://github.com/openattic/ceph/pull/1#issuecomment-359410074

**Comment by LenzGr** [comment]
@rjfd OK, will change this accordingly and rebase my PR

File: src/pybind/mgr/openattic/module.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+class Module(MgrModule):
+
+    """
+    Hello.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(Module, self).__init__(*args, **kwargs)
```
Comment (LenzGr) [inline src/pybind/mgr/openattic/module.py:24]: I currently get a `W0235:Useless super delegation in method '__init__'` - not sure if that can be ignored?

Comment (rjfd) [reply src/pybind/mgr/openattic/module.py:24]: yes, you can ignore this. We need to be able to make pylint to know the existence of `mgr_module.py` file where the `MgrModule` class is declared.

**Review by ricardoasmarques — APPROVED** [review]
It works

**Review by rjfd — APPROVED** [review]
lgtm

**Pull request merged**